CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.22.0/0.47.0/0.16.0/0.2.0] - 2024-01-18
+
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
```

Comment:
This line must be moved to unreleased.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4827/commits/1cd1f7075b29a312e214e7f96466ad81cc609415

Reply:
I misunderstood this comment, and didn't notice you had pushed the change you intended.  I reverted it.